### DISCUSSION TITLE
Lepton ghost removal

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#42] [Comments_HIG_23_001] [answered]
@jialin-guo1 Please confirm if the ghost removal is not rejecting any events.

You can see the cut-flow table that you provided me earlier:

This is using M1000 signal sample.

| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           |  197000.00 | 167.18 |
| passed_trig                       |  187742.00 |  -4.70 |
| passed_n_2leps                    |  153719.00 | -18.12 |
| passed_flavor_charge              |  152717.00 |  -0.65 |
| passed_lepZpt_40_25               |  147278.00 |  -3.56 |
| passed_dR_lilj0point2             |  147278.00 |   0.00 |
| passed_Mll_4                      |  147275.00 |  -0.00 |
| passed_lepisolation               |  145438.00 |  -1.25 |
| passed_leptightID                 |  145083.00 |  -0.24 |
| passed_lepZmass40_180             |  139912.00 |  -3.56 |
| passed_lepZ                       |  139877.00 |  -0.03 |
| passed_lepclean                   |  138444.00 |  -1.02 |
| passed_jetpt30                    |  138444.00 |   0.00 |
| passed_nmergedjets_lepclean       |  136318.00 |  -1.54 |
| passed_mergedjetsPt200            |  136318.00 |   0.00 |
| passed_mergedjetEta2opint4        |  135786.00 |  -0.39 |
| passed_jeteta2opint4              |  127203.00 |  -6.32 |
| passed_dijetpt100                 |  125433.00 |  -1.39 |
| passed_mergedmass_40_180          |  120591.00 |  -3.86 |
| passed_MergedolonlyHiggs          |  109317.00 |  -9.35 |
| passed_ak4jetZm_40_180            |   99384.00 |  -9.09 |
| passed_particleNetZvsQCD0opint9   |   73733.00 | -25.81 |

**Comment by ram1123** [comment]
- [ ] Also, we need the same cut-flow table for the Z+jets background.

**Reply by jialin-guo1** [reply]
in Z+jet:

| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           | 79621328.00 | 164457.88 |
| passed_trig                       | 48647168.00 | -38.90 |
| passed_n_2ak4jets                 | 48647168.00 |   0.00 |
| passed_n_2leps                    | 31044042.00 | -36.19 |
| passed_flavor_charge              | 30804300.00 |  -0.77 |
| passed_lepZpt_40_25               | 21558056.00 | -30.02 |
| passed_dR_lilj0point2             | 21557888.00 |  -0.00 |
| passed_Mll_4                      | 21557544.00 |  -0.00 |
| passed_lepisolation               | 20953628.00 |  -2.80 |
| passed_leptightID                 | 20788496.00 |  -0.79 |
| passed_lepZmass40_180             | 18418404.00 | -11.40 |
| passed_lepZ                       | 18402514.00 |  -0.09 |
| passed_lepclean                   | 17035504.00 |  -7.43 |
| passed_jetpt30                    | 17035504.00 |   0.00 |
| passed_jeteta2opint4              | 12855582.00 | -24.54 |
| passed_dijetpt100                 | 10616481.00 | -17.42 |
| passed_ak4jetZm_40_180            | 6482624.00 | -38.94 |
| passed_nmergedjets_lepclean       | 3872659.00 | -40.26 |
| passed_mergedjetsPt200            | 3872659.00 |   0.00 |
| passed_mergedjetEta2opint4        | 3830996.00 |  -1.08 |
| passed_mergedmass_40_180          |  788321.00 | -79.42 |
| passed_MergedolonlyHiggs          |  527509.00 | -33.08 |
| passed_particleNetZvsQCD0opint9   |   48385.00 | -90.83 |


passed_dR_lilj0point2  also make 0% changes

**Reply by ram1123** [reply]
For this passed_dR_lilj0point2 can you get the number. You will see if you increase the precision. As I can see that there is change in the number.

**Reply by jialin-guo1** [reply]
| Cut string name                   | left nEvents | Change (%) |
|-----------------------------------|------------|---------|
| nEvents                           | 79621328.0000 | 164457.8754 |
| passed_trig                       | 48647168.0000 | -38.9018 |
| passed_n_2ak4jets                 | 48647168.0000 | 0.0000 |
| passed_n_2leps                    | 31044042.0000 | -36.1853 |
| passed_flavor_charge              | 30804300.0000 | -0.7723 |
| passed_lepZpt_40_25               | 21558056.0000 | -30.0161 |
| passed_dR_lilj0point2             | 21557888.0000 | -0.0008 |
| passed_Mll_4                      | 21557544.0000 | -0.0016 |
| passed_lepisolation               | 20953628.0000 | -2.8014 |
| passed_leptightID                 | 20788496.0000 | -0.7881 |
| passed_lepZmass40_180             | 18418404.0000 | -11.4010 |
| passed_lepZ                       | 18402514.0000 | -0.0863 |
| passed_lepclean                   | 17035504.0000 | -7.4284 |
| passed_jetpt30                    | 17035504.0000 | 0.0000 |
| passed_jeteta2opint4              | 12855582.0000 | -24.5365 |
| passed_dijetpt100                 | 10616481.0000 | -17.4173 |
| passed_ak4jetZm_40_180            | 6482624.0000 | -38.9381 |
| passed_nmergedjets_lepclean       | 3872659.0000 | -40.2609 |
| passed_mergedjetsPt200            | 3872659.0000 | 0.0000 |
| passed_mergedjetEta2opint4        | 3830996.0000 | -1.0758 |
| passed_mergedmass_40_180          | 788321.0000 | -79.4226 |
| passed_MergedolonlyHiggs          | 527509.0000 | -33.0845 |
| passed_particleNetZvsQCD0opint9   | 48385.0000 | -90.8276 |

yes you are right. but the change is -0.0008%

**Reply by ram1123** [reply]
This means it reduces some of background events. But not the signal events, which is good.

**Reply by ram1123** [reply, marked as answer]
Conclusion:

0% removal of signal
0.0008% removal of Z+jet background using the ghost removal.

Need to think why, as this cut is looser than the isolation cut. So, this cut should not have any impact.
